client/plots/test/expclust.gdc.spec.js

```diff
@@ -241,15 +214,66 @@ tape('launch gene exp clustering using runproteinpaint with launchGdcHierCluster
 		},
 		settings: {
 			hierCluster: {
-				maxGenes: 50
+				maxGenes: maxGeneCount
 			}
 		},
 		termgroups: [
 			{
 				name: 'Variables',
-				lst: [{ id: 'case.disease_type' }, { id: 'case.primary_site' }, { id: 'case.demographic.gender' }]
+				lst: dictTerms
+			}
+		],
+		opts: {
```

Comment:
@siosonel this postrender callback via runpp(opts.hierCluster) will execute early before map is loaded. could you suggest a fix?